deploy/helm/sumologic/values.yaml

```diff
@@ -2769,7 +2769,38 @@ otelcol:
         extensions:
           health_check: {}
         exporters:
-          sumologic:
+          sumologic/containers:
+            log_format: json
+            endpoint: ${SUMO_ENDPOINT_DEFAULT_LOGS_SOURCE}
+            source_name: "%{_sourceName}"
+            source_category: "%{_sourceCategory}"
+            source_host: "%{_sourceHost}"
+            sending_queue:
+              enabled: true
+            metadata_attributes:
+              ## Required to be used in headers
+              - _sourceCategory
+              - _sourceHost
+              - _sourceName
+              ## Attributes to be send as fields
+              - _collector
+              - k8s.*
+              - pod_.*
+              - namespace_.*
+              ## ToDo: fix below duplication after https://github.com/SumoLogic/sumologic-otel-collector/pull/245 being resolved
```

Comment:
By `fix` we mean .... remove all below listed entries/regexes?

Reply:
yes

Reply:
So.... Can we change the comment to say so :) ?

Reply:
hmm..., maybe